specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1056,7 +1104,96 @@
       }
     }
   },
+  "x-ms-paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}?export=true": {
```

Comment:
If your request/response differs if an optional query parameter is provided, please use extension `x-ms-path`: "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}.

Reply:
@jianghaolu, I am already using the correct extension [x-ms-paths](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-paths). There is no extension `x-ms-path`

Reply:
@olydis, can you take a look at the PR? This is the second iteration of https://github.com/Azure/azure-rest-api-specs/pull/1554

Reply:
@olydis, can you take a look at the PR? This is the second iteration of https://github.com/Azure/azure-rest-api-specs/pull/1554

Reply:
You're right. My bad.